forc/src/cli/commands/build.rs

```diff
@@ -21,13 +21,18 @@ pub struct Command {
     /// Path to the project, if not specified, current working directory will be used.
     #[clap(short, long)]
     pub path: Option<String>,
-    /// Whether to compile to bytecode (false) or to print out the generated ASM (true).
+    /// Print the finalized ASM.
+    ///
+    /// This is the state of the ASM with registers allocated and optimisations applied.
```

Comment:
```suggestion
    /// This is the state of the ASM with registers allocated and optimizations applied.
```

American English perhaps? Seems good otherwise